hdx_hapi/db/models/views/all_views.py

```diff
@@ -226,26 +259,23 @@ class OperationalPresenceView(Base):
     admin2_is_unspecified: Mapped[bool] = column_property(operational_presence_view.c.admin2_is_unspecified)
     admin1_ref: Mapped[int] = column_property(operational_presence_view.c.admin1_ref)
 
-
 class OrgTypeView(Base):
     __table__ = org_type_view
     code: Mapped[str] = column_property(org_type_view.c.code)
     description: Mapped[str] = column_property(org_type_view.c.description)
 
-
 class OrgView(Base):
     __table__ = org_view
     acronym: Mapped[str] = column_property(org_view.c.acronym)
     name: Mapped[str] = column_property(org_view.c.name)
     org_type_code: Mapped[str] = column_property(org_view.c.org_type_code)
     org_type_description: Mapped[str] = column_property(org_view.c.org_type_description)
 
-
 class PopulationView(Base):
     __table__ = population_view
     resource_hdx_id: Mapped[str] = column_property(population_view.c.resource_hdx_id)
     admin2_ref: Mapped[int] = column_property(population_view.c.admin2_ref)
-    gender: Mapped[Gender] = column_property(population_view.c.gender)
+    gender: Mapped[str] = column_property(population_view.c.gender)
```

Comment:
Similar as above. I think it's easier to just use the `all_views.py` from `dev`

Reply:
These are fixed in c6c4800d307df572c543ef87fe9127cdf7e30920

I don't think my code is able to pick up the enum types, so I just copied over `all_views.py` from `dev` as you suggested.